ivy_tests/test_ivy/helpers.py

```diff
@@ -2373,9 +2373,9 @@ def dtype_and_values(
     num_arrays=1,
     min_value=None,
     max_value=None,
-    large_value_safety_factor=1.1,
-    small_value_safety_factor=1.1,
-    max_op="divide",
+    large_abs_safety_factor=1,
```

Comment:
I would suggest we keep the default value above `1`, otherwise many tests start to fail I believe, with values being generated right at the representable limit. Did you observe different behavior when running the tests? Do they all seem to work okay with a value of `1`?